LiskHQ/lisk
# Consistent usage of es6 features in logic implementation - Closes #1656

### What was the problem?
Some of the logic implementations are not inline with the es6 features.
### How did I fix it?
Use of es6 features in logic repo to make sure consistency and making use of the latest features.
### How to test it?
run test `npm run test -- mocha:untagged:unit` to ensure all the tests are working.
### Review checklist

* The PR solves #1656 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
